app/scripts/templates/confirm_sign_in.mustache

```diff
@@ -0,0 +1,29 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-confirm-header">{{#t}}Confirm this sign-in{{/t}}</h1>
```

Comment:
We'll have to add a new noun between `confirm` and `header` so our functional tests have a unique ID to test against that's not shared with other screens. `token` of we go with that [up here](https://github.com/mozilla/fxa-content-server/pull/3651/files#diff-44602b97da20196207f66f8fb0df0d8bR111)?